frontend/libs/erxes-ui/src/modules/record-field/components/NumberField.tsx

```
@@ -0,0 +1,64 @@
+      >
+        <RecordTableCellTrigger {...props} ref={ref}>
+          {children}
+          <TextOverflowTooltip value={editingValue.toString() ?? placeholder} />
+        </RecordTableCellTrigger>
+        <RecordTableCellContent asChild>
+          <form onSubmit={handleAction}>
+            <Input
+              value={editingValue.toString()}
+              onChange={(e) => {
+                setEditingValue(Number(e.target.value));
+                onValueChange && onValueChange(Number(e.target.value));
```

ESLint

[error] Expected an assignment or function call and instead saw an expression.

(@typescript-eslint/no-unused-expressions)